common/src/main/java/com/microsoft/hydralab/common/util/ShellUtils.java

```diff
@@ -14,17 +16,18 @@ public class ShellUtils {
     private static String[] getFullCommand(String command)
     {
         String shellProcess = "";
-        String argName = "";
+        String args = "";
 
         if (isConnectedToWindowsOS) {
+            // Add execution policy to ensure powershell can run on most of Windows devices
             shellProcess = POWER_SHELL_PATH;
-            argName = "-Command";
+            args = "powershell -ExecutionPolicy Unrestricted -NoProfile -Command";
```

Comment:
Please pass the same args for `killProcessByCommandStr()` which dose not call the function `getFullCommand()`

Reply:
Fixed and verified